### PR TITLE
Upstream tag fix

### PR DESCRIPTION
**Pull request by jackhodgkiss**
This fixes the two issues regarding hidden tags and unwanted upstream tags causing undesired output from the script.

### CODE REVIEW

File: os-downstream-tag.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 import sys
 
 
-DEFAULT_RELEASE = os.environ.get("DEFAULT_RELEASE", "wallaby")
+DEFAULT_RELEASE = os.environ.get("DEFAULT_RELEASE", "victoria")
```
Comment (markgoddard) [inline os-downstream-tag.py]: Unrelated?

File: os-downstream-tag.py [hunk 2]
```diff
@@ -30,11 +30,17 @@ def fetch(remote):
 
 
 def most_recent_tag(ref):
-    cmd = ["git", "describe", ref, "--abbrev=0", "--tags"]
+    cmd = ["git", "describe", ref, "--exclude=*[a-z]", "--abbrev=0", "--tags"]
```
Comment (markgoddard) [inline os-downstream-tag.py:33]: This may exclude alpha and beta style releases (`1.2.3-alpha`), but I don't think that's an issue right now.

**Review by markgoddard — APPROVED** [review]

File: os-downstream-tag.py (hunk 1 above)
Comment (jackhodgkiss) [reply os-downstream-tag.py]: Reverted back to wallaby. It was done whilst I was testing the issue which was present on the victoria branch.

File: os-downstream-tag.py (hunk 2 above)
Comment (jackhodgkiss) [reply os-downstream-tag.py:33]: --exclude and --match are not very flexible would probably better to use grep in the future.
